packages/react-components/src/trace-explorer/trace-explorer-views-widget.tsx

```diff
@@ -44,54 +54,77 @@ export class ReactAvailableViewsWidget extends React.Component<ReactAvailableVie
     render(): React.ReactNode {
         return (
             <div className="trace-explorer-views">
-                <AvailableViewsComponent
-                    traceID={this._selectedExperiment?.UUID}
-                    outputDescriptors={this.state.availableOutputDescriptors}
-                    onContextMenuEvent={this.handleContextMenuEvent}
-                    onOutputClicked={this.handleOutputClicked}
-                    highlightAfterSelection={true}
-                ></AvailableViewsComponent>
+                <div className="trace-explorer-panel-content">{this.renderOutputs()}</div>
             </div>
         );
     }
 
-    protected handleOutputClicked = (outputDescriptor: OutputDescriptor): void =>
-        this.doHandleOutputClicked(outputDescriptor);
-    protected handleContextMenuEvent = (
-        e: React.MouseEvent<HTMLDivElement>,
-        output: OutputDescriptor | undefined
-    ): void => this.doHandleContextMenuEvent(e, output);
+    private renderOutputs() {
+        if (this.state.treeNodes) {
+            return (
+                <div className="scrollable avail-views-table">
+                    <FilterTree
```

Comment:
should we move this into AvailableViewComponent? Then the Overview menu item to select the xy output would have it as well.